proof/append_test.go

```diff
@@ -23,7 +23,7 @@ func genTaprootKeySpend(t testing.TB, privKey btcec.PrivateKey,
 
 	t.Helper()
 
-	virtualTxCopy := tapscript.VirtualTxWithInput(
+	virtualTxCopy := asset.VirtualTxWithInput(
```

Comment:
nit: looks like this should be in an earlier commit to allow all commits to compile on their own (can use `git rebase -i v0-3-0-staging --exec "make unit pkg=... case=_none_"` to test that, though it looks like the `v0-3-0staging` branch has a compilation issue already, so no priority, can do that before merging it into `main`)